avm/src/main.rs

```diff
@@ -167,10 +170,17 @@ fn main() {
             let agz_file = matches.value_of("AGZ_FILE").unwrap();
             let app_name = matches.value_of("app-name").map(String::from);
             let config_name = matches.value_of("config-name").map(String::from);
+            let files = matches.value_of("files");
+            let mut files_b64 = HashMap::new();
+            if let Some(files) = files {
+              let names = files.split(",");
```

Comment:
I was going to suggest if a `trim` would be necessary, but one cannot add spaces on bash command, right?

Reply:
It's theoretically possible but you have to intentionally bypass bash's standard parsing with backslashes or quotes.